tests/e2e/test_diffusion.py

```diff
@@ -590,10 +619,6 @@ def sd15_std_sde(
 def sd15_std_float16(
     text_encoder_weights: Path, lda_weights: Path, unet_weights_std: Path, test_device: torch.device
 ) -> StableDiffusion_1:
-    if test_device.type == "cpu":
-        warn("not running on CPU, skipping")
-        pytest.skip()
-
```

Comment:
Any reason to remove these lines? This is probably a debug leftover. Could you please undo these changes?

Reply:
oh! that was a typo